tools/autograd/gen_variable_type.py

```diff
@@ -41,7 +41,7 @@
 DONT_RECORD_TRACE = {
     'convolution', 'conv1d', 'conv2d', 'conv3d', 'conv_transpose1d',
     'conv_transpose2d', 'conv_transpose3d', 'lstm_cell', 'gru_cell',
-    'rnn_tanh_cell', 'rnn_relu_cell', 'linear',
+    'rnn_tanh_cell', 'rnn_relu_cell',
```

Comment:
so it's okay to trace this now?  For changes like this I'd expect them to be broken out by themselves and explained.

Reply:
Not sure, I make this change just to make linear shows up in the autograd profiler. Let's see if it breaks CI, if yes we can add it back to the blacklist.

Reply:
it likely would break onnx export, no?

Reply:
It's "fixable" - I have added an onnx symbolic for linear